runconfig/config.go

```diff
@@ -30,6 +31,7 @@ type Config struct {
 	Cmd             *stringutils.StrSlice // Command to run when starting the container
 	Image           string                // Name of the image as it was passed by the operator (eg. could be symbolic)
 	Volumes         map[string]struct{}   // List of volumes (mounts) used for the container
+	CLIVolumeSpecs  map[string]struct{}   // CLI-passed list of volumes (mounts) used for the container
```

Comment:
Ha! Sorry ;) @tiborvass - give me a name you guys agree on and I'll update it ASAP.